### DISCUSSION TITLE
Sort control direction

### DISCUSSION THREAD

**Discussion by salascz** [microsoft-search/pnp-modern-search#2064] [Q&A]
Hello,
I like the Sort control introduced in April 2022 release. 

Is there a way, how to change the default sort direction? (The direction seems to be disabled when using "User sort")

Use case: I would like to sort results in descending order (e.g. Newest results or results with the largest number of visits `ViewsLifeTime`)

Direction settings does not seem to be applied:
![image](https://user-images.githubusercontent.com/5662939/166677021-f5ad2b38-402e-48e5-9f07-01947af72de8.png)

Thanks

**Comment by FranckyC** [comment]
Hi @salascz , short answer: no. the default direction only applies for default sort fields and the sort direction for user sort field is always ascending.

**Comment by salascz** [comment]
Is there any reason behind? I understand there might be an issue with UI (with arrows next to the combobox control, maybe there could be an option to hide directional arrows?)

I am trying to figure out, how to use this control... 

My original intention was to have filters like:

* most relevant (technically: Rank descending)
* newest (technically: Created descending)
* most viewed (technically: ViewsLifeTime descending)

**Comment by DavorSukeri** [comment]
This is something I have wondered since I the first days I started using PnP Modern Search and now one client asked the same question so I decided to come ask here.

So the same question:

In most cases I have set default sort to Descending even if there were multiple different sort options available as User sort. They also are mostly used in Descending order. For example:

We have created a "statistics" page that lists all the pages, news, etc (those are added as verticals) and we display ViewsLifetime and ViewsLifetimeUniqueUsers properties in the Search results card (custom template). Default sort is Views and the sort order is Descending.
Now if I want to sort by the ViewsLifeTimeUniqueUsers I select it from the sortmenu and it defaults to ascending and I have click the arrow to change it to descending.

Wouldn't it be great if the User sort order would default to same direction with Default sort?

**Reply by wobba** [reply]
@DavorSukeri probably. If you can have someone PR a change for this we will merge it.

**Reply by DavorSukeri** [reply]
> @DavorSukeri probably. If you can have someone PR a change for this we will merge it.

Ok,

I think it depends if customer wants to pay for us to learn the solution/code, make and test the change internally and then PR the change to here or if one of developers have time and are willing to do the research and testing as non-billable work.
